tasks/main.yml

```diff
@@ -43,6 +50,50 @@
   notify: restart influxdb
   tags: [influxdb]
 
+- name: Disable HTTP query logging for influxdb
+  block:
+    - lineinfile:
+        path: /etc/influxdb/influxdb.conf
+        line: '  log-enabled = false'
+        insertafter: '\[http\]'
+        state: present
+    - lineinfile:
+        path: /etc/influxdb/influxdb.conf
+        line: '  query-log-enabled = false'
+        insertafter: '\[data\]'
+        state: present
+  when: ansible_ow_influxdb_http_logging is false
+  tags: [influxdb]
+
+- name: Enable HTTP query logging for influxdb
+  block:
+    - lineinfile:
+        path: /etc/influxdb/influxdb.conf
+        line: '  log-enabled = false'
+        insertafter: '\[http\]'
+        state: absent
+    - lineinfile:
+        path: /etc/influxdb/influxdb.conf
+        line: '  query-log-enabled = false'
+        insertafter: '\[data\]'
+        state: absent
+  when: ansible_ow_influxdb_http_logging is true
+  tags: [influxdb]
```

Comment:
This exists to revert the changes of previous task, otherwise it won't change anything if users first run with default settings and then realises that they forget to set `ansible_ow_influxdb_http_logging` to `true`.